### DISCUSSION TITLE
[aws-lambda plugin] Kong is not using the IAM role attached to service account of EKS.

### DISCUSSION THREAD

**Discussion by jyoti-samal-bh** [Kong/kong#10434] [Help]
Kong API gateway is not using the IAM role attached to service account of  EKS. Instead its using the IAM role attached to the EC2 instance where the EKS node is running. We tried to use assume_role_arn, but in that case also, its not working. As it gets stuck at line "Trying to assume the role". Please help.

**Comment by ammar-elsabe** [comment]
hi did you manage to solve this?
